documentation/release_4.0.htm

```diff
@@ -134,6 +135,9 @@ <h3>New functionality</h3>
 <ul>
 <li><tt>KOSMAPOSL</tt> STIR now allows to use the hybrid kernel method described in the following paper: "Hybrid PET-MR list-mode kernelized expectation maximization reconstruction",
   Inverse Problems, 2019, DOI: https://doi.org/10.1088/1361-6420/ab013f.</li>
+<li><tt>KOSMAPOSL</tt> also allows to use multiple anatomical images. The method is described in "Multiplexing Kernelized Expectation Maximization
+Reconstruction for PET-MR", NSS/MIC proceedings, 2018, DOI: https://doi.org/10.1109/NSSMIC.2018.8824312.
```

Comment:
acutally, can't have plain quotes in HTML. Could use `&quot;` or make it italic instead (`<i>bla bla</i>`)